### ISSUE TITLE
Text component 'accessibilityRole` property not being read

### ISSUE THREAD

**Issue by YajurG** [microsoft/react-native-windows#11563]
### Problem Description

I've been trying to set the accessibilityRole property of a <Text> component and it is simply not being read by the TextViewManager.  When debugging through VS 2022, I set a breakpoint inside the conditional checking for the 'accessibilityRole' property which is never hit. Not sure about the root reason for this issue as the other Text properties are being read correctly. 

While wrapping the text component in a `<View>` and setting the accessibilityRole of the view can be a workaround, the View component does not have all the same accessibilityRole values (such as 'header) which does not make it a proper solution.

Example code snippet:

```
<View> 
  <Text 
    accessible 
    accessibilityRole="header" 
    numberOfLines={1} 
    style={[styles.title, {color: 'grey'}]}>
    Expander 
  </Text>
</View>
```

In the above example, the debugger showed that the 'accessible' and 'numberOfLines' properties were being read, but the accessibilityRole was not - when stepping through and continuing in the debugger the accessibilityRole property did not show up. 

### Steps To Reproduce

1. Copy above example into Playground simple example file
2. Set breakpoint in VS 2022 on line 310 of TextViewManager.cpp
3. Continue through different property names, and note if accessibilityRole does not show up.

### Expected Results

When accessibilityRole is set, the propertyName should be read through the set breakpoint, but it instead does not appear. This means the accessibilityRole property is not being read by the TextViewManager, which means it cannot be used within a Text component.

### CLI version

12.0.0-alpha.0

### Environment

```markdown
System:
  OS: Windows 10 10.0.22621
  CPU: "(24) x64 AMD Ryzen Threadripper PRO 3945WX 12-Cores     "
  Memory: 46.35 GB / 63.86 GB
Binaries:
  Node:
    version: 18.12.1
    path: C:\Program Files\nodejs\node.EXE
  Yarn:
    version: 1.22.19
    path: C:\Program Files (x86)\Yarn\bin\yarn.CMD
  npm:
    version: 8.19.2
    path: C:\Program Files\nodejs\npm.CMD
  Watchman: Not Found
SDKs:
  Android SDK: Not Found
  Windows SDK:
    AllowDevelopmentWithoutDevLicense: Enabled
    AllowAllTrustedApps: Enabled
    Versions:
      - 10.0.18362.0
      - 10.0.19041.0
      - 10.0.22000.0
      - 10.0.22621.0
IDEs:
  Android Studio: AI-213.7172.25.2113.9123335
  Visual Studio:
    - 17.5.33530.505 (Visual Studio Enterprise 2022)
    - 16.11.33529.622 (Visual Studio Enterprise 2019)
Languages:
  Java:
    version: 18.0.2.1
    path: C:\Program Files\Common Files\Oracle\Java\javapath\javac.EXE
  Ruby: Not Found
npmPackages:
  "@react-native-community/cli": Not Found
  react: Not Found
  react-native: Not Found
  react-native-windows: Not Found
npmGlobalPackages:
  "*react-native*": Not Found
Android:
  hermesEnabled: Not found
  newArchEnabled: Not found
iOS:
  hermesEnabled: Not found
  newArchEnabled: Not found
```

### Target Platform Version

None

### Visual Studio Version

Visual Studio 2022

### Build Configuration

Debug

### Snack, code example, screenshot, or link to a repository

```
<View> 
  <Text 
    accessible 
    accessibilityRole="header" 
    numberOfLines={1} 
    style={[styles.title, {color: 'grey'}]}>
    Expander 
  </Text>
</View>
```

**Comment by chrisglein** [comment]
Implementation for this is in FrameworkViewManager, and only picked up for View panels. Text isn't picking these up.
See if accessibilityRole is being exposed properly with the native props on TextViewManager with the right type.

**Comment by YajurG** [comment]
Tested this in 0.70 and accessibilityRole property was being read successfully - narrator read out the Text component as 'Heading level 2'.

**Comment by YajurG** [comment]
Looks like Andrew's PR fixed this issue and this can be closed
